akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcClusterSpec.scala

```diff
@@ -20,6 +20,7 @@ class MultiDcSpecConfig(crossDcConnections: Int = 5) extends MultiNodeConfig {
   commonConfig(ConfigFactory.parseString(
     s"""
       akka.loglevel = INFO
+      akka.coordinated-shutdown.terminate-actor-system = off
```

Comment:
we have this in src/test/resources/reference.conf for the reason you describe, so it's strange if this test is not using that. it's a bit dangerous to override config with another reference.conf like that because it depends on classpath ordering, but we have never seen a problem with that for the test/resources in same project.

Reply:
Ah, hadn't seen it there. I wonder if the order is still guaranteed as this is in the test tree rather than the multi-jvm tree.